config/webpack/rscWebpackConfig.js

```
@@ -0,0 +1,112 @@
+  rscConfig.plugins = rscConfig.plugins.filter(
+    (plugin) =>
+      plugin.constructor.name !== 'WebpackAssetsManifest' &&
+      plugin.constructor.name !== 'MiniCssExtractPlugin' &&
+      plugin.constructor.name !== 'CssExtractRspackPlugin' &&
+      plugin.constructor.name !== 'ForkTsCheckerWebpackPlugin',
+  );
+
+  // Remove CSS extraction loaders from style rules
+  rscConfig.module.rules.forEach((rule) => {
+    if (Array.isArray(rule.use)) {
+      rule.use = rule.use.filter((item) => {
```

ESLint

[error] Assignment to property of function parameter 'rule'.

(no-param-reassign)